doc/mgr/dashboard.rst

```diff
@@ -378,6 +378,27 @@ The available iSCSI gateways must be defined using the following commands::
 Enabling the Embedding of Grafana Dashboards
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+`Grafana <https://grafana.com/>`_ needs to request data from `Prometheus
+<https://prometheus.io/>`_. The Grafana dashboards we provide contain queries
+that are specific to Prometheus. Although Grafana can have other data sources,
+the Grafana dashboards only support Prometheus as data source. The the Ceph
+:ref:`mgr-prometheus` also only exports its data in the Prometheus' common
+format. The Grafana Dashboards also rely on metric names from the Prometheus
+module and `Node exporter <https://prometheus.io/docs/guides/node-exporter/>`_.
+The Node exporter is a separate application that provides machine metrics.
+
+.. note::
+
+  Please note that Prometheus' security model presumes that untrusted users
+  have access to the Prometheus HTTP endpoint and logs. They have access to
+  all time series information contained in the database, plus a variety of
```

Comment:
Same question as above regarding the meaning of "all time series information"?

Reply:
Same answer as above :)